core/arch/arm/include/sm/optee_smc.h

```diff
@@ -290,6 +290,12 @@
 #define OPTEE_SMC_SEC_CAP_HAVE_RESERVED_SHM	(1 << 0)
 /* Secure world can communicate via previously unregistered shared memory */
 #define OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM	(1 << 1)
+/*
```

Comment:
Hi, @stuyoder. I did exactly this. I decided not to touch existing `OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM`. Instead I added new `OPTEE_SMC_SEC_CAP_DYNAMIC_SHM`. I think, that this name better describes this capability.